docs/openapi-ts/configuration.md

```diff
@@ -105,28 +105,68 @@ You might not need a `node` client. Fetch API is [experimental](https://nodejs.o
 
 By default, `openapi-ts` will automatically format your client according to your project configuration. To disable automatic formatting, set `format` to false
 
-```js{2}
+::: code-group
+
+```js{2} [prettier (default)]
+export default {
+  format: 'prettier',
+  input: 'path/to/openapi.json',
+  output: 'src/client',
+}
+```
+
+```js{2} [biome]
+export default {
+  format: 'biome',
+  input: 'path/to/openapi.json',
+  output: 'src/client',
+}
+```
+
+```js{2} [off]
```

Comment:
Can you make this default too? I think we're in a good place now to turn it off and end up with a reasonable output + it will be consistent with linting

```suggestion
```js{2} [disabled]
```